### ISSUE TITLE
Report concept set presences that are 0

### ISSUE THREAD

**Issue by alondhe** [OHDSI/ClinicalCharacteristics#11]
When using add<DomainId>Presence, we currently lose line items for concept sets that have no presence during the time window. 

For table reporting, it's useful to keep these in with N = 0 and Percent = 0.

**Comment by mdlavallee92** [comment]
I will try to fix this but this maybe an artifact of summarizing in SQL. If there is no count for a concept set and I do a GROUP BY, it won't use an item that is not present. 

Can add a post-processing step to add a placeholder for concept set with no presence or are below a minimum reportable value (say <5). Agree something should be done for sake of completeness

**Comment by alondhe** [comment]
Agreed, doesn't need to be during processing. But I think it'd be great if in the add<Domain>Presence functions, we keep a parameter like 'reportZero' and then in the finished categorical result, we get those items.

**Comment by mdlavallee92** [comment]
adding to v1 milestone
